taskfile.yaml

```
@@ -26,15 +26,20 @@ tasks:
     silent: false
 
   rebuild:
+    deps: [clean-db, clean-modcache, clean-cache]
     cmds:
-      - cmd: go clean --modcache
-      - cmd: go clean -cache
       - task: tidy
       - task: verify
       - task: build
         vars: { TASK_OPTIONS: "-a" }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)